projects/1.18-fabric/assets/interactic/interactic/lang/zh_cn.json

```
@@ -0,0 +1,26 @@
+{
+  "item.interactic.item_filter": "物品过滤器",
+  "death.attack.thrown_item": "%1$s was pummeled by %2$s using %3$s",
+  "death.attack.thrown_item.sword": "%1$s was executed by %2$s using %3$s",
+  "death.attack.thrown_item.axe": "%1$s felt a trickshot %2$s performed using %3$s",
+  "death.attack.thrown_item.shovel": "%1$s was whacked by %2$s using %3$s",
+  "death.attack.thrown_item.pickaxe": "%1$s was pierced by %2$s using %3$s",
+  "death.attack.thrown_item.hoe": "%1$s was reaped by %2$s using %3$s",
+
+  "text.autoconfig.interactic.title": "Interactic配置",
+  "text.autoconfig.interactic.category.global": "全局选项",
+  "text.autoconfig.interactic.category.client": "仅限客户端选项",
+
+  "key.interactic.pickup_item": "［Interactic］捡起物品",
+
+  "text.autoconfig.interactic.option.clientOnlyMode": "仅限客户端模式",
+  "text.autoconfig.interactic.option.rightClickPickup": "右键拾取",
+  "text.autoconfig.interactic.option.itemFilterEnabled": "物品过滤器",
+  "text.autoconfig.interactic.option.autoPickup": "自动拾取",
+  "text.autoconfig.interactic.option.fancyItemRendering": "平放物品渲染（渲染物品）",
+  "text.autoconfig.interactic.option.renderItemTooltips": "渲染物品工具提示框",
```

```suggestion
  "text.autoconfig.interactic.option.renderItemTooltips": "渲染物品提示框",
```
https://minecraft.fandom.com/zh/wiki/%E6%8F%90%E7%A4%BA%E6%A1%86
不需要把Tool单独翻译出来